.github/workflows/reusable-rosdoc2.yml

```diff
@@ -0,0 +1,38 @@
+name: rosdoc2
+# author: Christoph Froehlich <christoph.froehlich@ait.ac.at>
+
+on:
+  workflow_call:
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    env:
+      # this will be src/{repo-owner}/{repo-name}
+      path: src/${{ github.repository }}
+    steps:
+      - uses: ros-tooling/setup-ros@0.7.1
+      - name: Install rosdoc2 and deps
+        run: |
+          sudo apt-get update
+          sudo apt install -y python3-pip git doxygen graphviz
+          git clone https://github.com/ros-infrastructure/rosdoc2.git
+          cd rosdoc2
+          pip install --user --upgrade .
```

Comment:
I suggest using a virtualenv, since otherwise things [will break](https://stackoverflow.com/questions/75602063/pip-install-r-requirements-txt-is-failing-this-environment-is-externally-mana/75696359#75696359) on 24.04.

Reply:
I know and it is also not working with `act` in this way. But I never configured a virtualenv here, do you have a best-practice example for that? It doesn't seem to be that common yet.

Reply:
ok I think this is not a big deal, see last commit.